Makefile

```diff
@@ -94,33 +94,34 @@ e2e-teardown:
 .PHONY: azuredisk
 azuredisk:
 	if [ ! -d ./vendor ]; then dep ensure -vendor-only; fi
-	CGO_ENABLED=0 GOOS=linux go build -a -ldflags ${LDFLAGS} -o _output/azurediskplugin ./pkg/azurediskplugin
+	CGO_ENABLED=0 GOOS=linux go build -a -ldflags "${LDFLAGS}" -o _output/azurediskplugin ./pkg/azurediskplugin
 
 .PHONY: azuredisk-windows
 azuredisk-windows:
 	if [ ! -d ./vendor ]; then dep ensure -vendor-only; fi
-	CGO_ENABLED=0 GOOS=windows go build -a -ldflags ${LDFLAGS} -o _output/azurediskplugin.exe ./pkg/azurediskplugin
+	CGO_ENABLED=0 GOOS=windows go build -a -ldflags "${LDFLAGS}" -o _output/azurediskplugin.exe ./pkg/azurediskplugin
 
 .PHONY: container
-container: azuredisk
```

Comment:
make keep this, otherwise `make container` won't build linux binary

Reply:
Moved that inside the container now, so should build container regardless of local dev env